### PR TITLE
Movie page

### PR DESCRIPTION
**Pull request by spaceplesiosaur**
#### What's this PR do?

This adds the functionality and testing to the MoviePage component, so it includes the MovieBox and MovieCard components as well. 

It also adds the CSS, but since its my CSS it will probably need some help - feel free to lmk what needs to be corrected.

#### Where should the reviewer start?

The CSS probably needs the most help, but if you could check to see if there are any tests that I missed writing that would be great.  I don't think there's anything I need to refactor, but if there is, lmk!

#### How should this be manually tested?

You can pull down and run the tests, or we can just do it in person during work time.

#### Questions:

Totally clueless on how to format the user profile - sorry about that!

### CODE REVIEW

File: src/components/MoviesPage/__snapshots__/MoviesPage.test.js.snap [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  >
+    <UserProfile
+      0="T"
+      1="h"
+      2="e"
+      3=" "
+      4="R"
+      5="o"
+      6="c"
+      7="k"
```
Comment (RayRedGoose) [inline src/components/MoviesPage/__snapshots__/MoviesPage.test.js.snap]: I guess there is some error with rest operator when you  add props to UserProfile component

File: src/components/MoviesBox/MoviesBox.js [hunk 2]
```diff
@@ -5,14 +5,33 @@ import './MoviesBox.scss';
...
-    this.state = {}
+    this.state = {
+      cards: []
+    }
+  }
+
+  componentDidMount = () => {
+    fetch('https://swapi.co/api/films/')
+    .then(response => response.json())
+    .then(data => this.setState({cards: data.results.sort((a, b) => {return a.episode_id - b.episode_id})}))
```
Comment (RayRedGoose) [inline src/components/MoviesBox/MoviesBox.js]: Maybe add one more .then( ) to sort response array and return it before setState is invoked?

**Review by RayRedGoose — CHANGES_REQUESTED** [review]
Great work with adding functionality and testing! CSS need some changes and there is no snapshots for UserProfile and MoviesBox.

File: src/components/MoviesPage/__snapshots__/MoviesPage.test.js.snap (hunk 1 above)
Comment (spaceplesiosaur) [reply src/components/MoviesPage/__snapshots__/MoviesPage.test.js.snap]: Cool, fixed it with user={{name: "The Rock"}} as per your suggestion

File: src/components/MoviesBox/MoviesBox.js (hunk 2 above)
Comment (spaceplesiosaur) [reply src/components/MoviesBox/MoviesBox.js]: good call, just added it!

**Comment by spaceplesiosaur** [comment]
Thanks for fixing the CSS!

**Review by RayRedGoose — APPROVED** [review]
Nice job! I approve all changes.

**Pull request merged**